test/integration/collection-management/collection.test.js

```diff
@@ -535,81 +535,81 @@ describe('Collection', function () {
     });
   });
 
-  describe('(countDocuments)', function () {
```

Comment:
These tests pass locally but fail on CI so I just rewrote them. They also weren't testing anything in some cases since any type of count would return `0` if no documents were inserted.